internal/commands/vaultsecrets/secrets/secret_types.go

```diff
@@ -0,0 +1,7 @@
+package secrets
+
+const (
+	secretTypeKV       = "kv"
+	secretTypeDynamic  = "dynamic"
+	secretTypeRotating = "rotating"
```

Comment:
A string enum might be more fitting here.

Reply:
Have an example of what you're thinking? Seen a few different patterns for Go enums so might not guess correctly.